specification/automation/resource-manager/Microsoft.Automation/preview/2023-05-15-preview/examples/job/resumeJob.json

```diff
@@ -0,0 +1,12 @@
+{
+  "parameters": {
+    "subscriptionId": "51766542-3ed7-4a72-a187-0c8ab644ddab",
+    "resourceGroupName": "mygroup",
+    "automationAccountName": "ContoseAutomationAccount",
+    "jobName": "foo",
+    "api-version": "2023-05-15-preview"
+  },
+  "responses": {
+    "200": {}
```

Comment:
>  "200": {} [](http://example.com/codeflow?start=2&length=11)

200 should have a body. when there's no body it can be 204

Reply:
There has been no changes in Job Swagger and this is already shipped. Also from API also we only return 200 with empty body for stopJob, resumeJob, suspendJob operations changing it to 204 would result in breaking change.